verilog_style/exclude_filelist.f

```diff
@@ -0,0 +1,5 @@
+riscv_instr_cover_group.sv
```

Comment:
Can you add a comment why these files are excluded?

Reply:
Done